tests/helpers/test_config_validation.py

```diff
@@ -110,7 +110,7 @@ def test_platform_config():
         {'platform': 'mqtt', 'beer': 'yes'},
     )
     for value in options:
-        cv.PLATFORM_SCHEMA(value)
+        cv.PLATFORM_SCHEMA_BASE(value)
```

Comment:
I'm not so sure about this one.
What's the point of this test case?

Reply:
I think it's that the base platform schema should require `'platform' ` key, and allow extra.

Reply:
OK, that makes sense.